### DISCUSSION TITLE
Not fix

### DISCUSSION THREAD

**Discussion by zjssun** [Kvan7/Exiled-Exchange-2#217] [Q&A]
![屏幕截图 2024-12-31 234535](https://github.com/user-attachments/assets/3ab3d066-6504-4a36-a253-d65c79542e44)

**Comment by vagrant-soul** [comment]
#285
Waiting for decision on adoption @Kvan7

**Reply by Kvan7** [reply]
ah my bad, thought this was for general items not working in chinese
